en/02_Developer_Guides/00_Model/10_Versioning.md

```diff
@@ -156,16 +174,201 @@ Versioning only works if you are adding the extension to the base class. That is
+    ];
+
+    private static $extensions = [
+        Versioned::class,
+    ];
+}
+```
+
+### Controlling permissions to versioned DataObjects {#permissions}
+
+By default, `Versioned` will come out of the box with security extensions which restrict the visibility of objects in Draft ("Stage") or Archive viewing mode.
+
+[alert]
+As is standard practice, user code should always invoke `canView()` on any object before
+rendering it. DataLists do not filter on `canView()` automatically, so this must be
+done via user code. This can be achieved either by wrapping `<% if $canView %>;` in
+your template, or by implementing your visibility check in PHP.
+[/alert]
+
+#### Version specific _can_ methods {#permission-methods}
+
+Versioned DataObjects get additional permission check methods to verify what operation a `Member` is allowed to perform:
+
+* [`canPublish()`](api:SilverStripe\Versioned\Versioned::canPublish()): Determines if a given `Member` is allowed to publish the record
+* [`canUnpublish()`](api:SilverStripe\Versioned\Versioned::canUnpublish()) Determines if a given `Member` is allowed to unpublish the record
+* [`canArchive()`](api:SilverStripe\Versioned\Versioned::canArchive()) Determines if a given `Member` is allowed to archive the record
+* [`canViewStage()`](api:SilverStripe\Versioned\Versioned::canViewStage()) Determines if a given `Member` can view the latest version of this record on a specific stage.
+* [`canViewVersioned()`](api:SilverStripe\Versioned\Versioned::canViewVersioned()) Provides additional can view checks for versioned records. This is called by `canView()` and should not be called directly.
+
+These methods accept an optional `Member` argument. If not provided, they will assume you want to check the permission against the current `Member`. When performing a version operation on behalf of a `Member`, you'll probably want to use these methods to confirm they are authorised.
+
+[warning]
+Like with the base `can` permission checks, these checks are _not_ performed automatically when invoking the associated action via PHP. i.e. if you call `publishSingle()` on a record in your own code, Silverstripe CMS will _not_ check if the currently authenticated user has permission to publish the record. Make sure you are performing permission checks by calling these `can` methods before invoking the associated actions.
+[/warning]
+
+```php
+use SilverStripe\Security\Security;
+
+$record = MyRecord::get()->byID(99);
+if ($record->canPublish()) {
+    $record->publishRecursive();
+}
+```
+
+The `canViewStage()` method can be used to check if a Member can view a specific stage of a record. It should be invoked by user code to check if a record is visible in the given stage.
+
+```php
+use SilverStripe\Versioned\Versioned;
+
+// Check if `$member` can view the Live version of $record.
+$record->canViewStage(Versioned::LIVE, $member);
+
+// Check if `$member` can view the Stage version of $record.
```

Comment:
```suggestion
// Check if `$member` can view the Stage/Draft version of $record.
```
Wondering if it's worth using stage/draft for everything - would that reduce confusion?

Reply:
Depends on the reader. It would be just as easy to read that as "the Stage version or the Draft version"
I should have put quotes around "Stage" there, though. I'll fix that.